src/ontology/mondo-edit.obo

```diff
@@ -526302,14 +526302,21 @@ property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/
 
 [Term]
 id: MONDO:0800306
-name: epilepsy, progressive myoclonic, 2b
+name: myoclonic epilepsy of Lafora 2
+def: "Any Lafora disease in which the cause of the disease is a variation in the NHLRC1 gene." [MONDO:patterns/disease_series_by_gene]
 subset: gard_rare {source="MONDO:GARD"}
 subset: inferred_rare
 subset: rare
+synonym: "epilepsy, progressive myoclonic, 2B" EXACT [OMIM:620681]
 synonym: "EPM2B" EXACT ABBREVIATION [OMIM:254780]
-xref: OMIM:254780 {source="MONDO:includedEntryInOMIM"}
```

Comment:
Why was this removed? Is this [subset](https://mondo.readthedocs.io/en/stable/editors-guide/f-entities/#subsets) still being used in genera?

Reply:
yes, it is still being used but it is no longer the case that this OMIM is an included entry in OMIM on this class